src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -469,6 +469,44 @@ OutRelationRowLock(OUTFUNC_ARGS)
 	WRITE_ENUM_FIELD(rowLockStrength, LockClauseStrength);
 }
 
+static void WriteTaskQuery(OUTFUNC_ARGS) {
+	WRITE_LOCALS(Task);
+	WRITE_NODE_TYPE("TASKQUERY");
```

Comment:
Shouldn't we remove these two lines?

Reply:
We should remove the second one, but WRITE_LOCALS(Task) expands to:
`const Task *node = (const Task *) raw_node`. As we need the Task* for the following lines, it is necessary. 

An alternative would be to send *Task as a parameter, but in that case we also need to send StringInfo str (We need that for basically each Write method). Using OUTFUNC_ARGS as a parameter seemed simpler and more consistent.